src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/AbstractRecordCollection.java

```diff
@@ -178,8 +180,12 @@ protected RECORD createRecord(final DataLine dataLine) {
         }
 
         private SAMFileHeader getHeader() throws FileNotFoundException {
-            final LineReader lineReader = new BufferedLineReader(new FileInputStream(file));
-            return new SAMTextHeaderCodec().decode(lineReader, getSource());
+            try {
+                final LineReader lineReader = new BufferedLineReader(Files.newInputStream(path));
+                return new SAMTextHeaderCodec().decode(lineReader, getSource());
+            } catch (IOException ex) {
```

Comment:
We typically make the final explicit (probably for no practical reason other than style).

Reply:
Thank you for the compliment! I'm guessing you mean the `IOException` variable? Done.